addon/components/create-overlay.js

```diff
@@ -38,16 +38,16 @@ export default Component.extend({
     }
   },
 
-  _translation(targetRect, ownRect, currentTransform) {
-    return `translateX(${targetRect.left - ownRect.left + currentTransform.tx}px) translateY(${targetRect.top - ownRect.top + currentTransform.ty}px)`;
+  _translation(targetRect, ownRect, currentTransform, oversized) {
+    return `translateX(${targetRect.left - ownRect.left + currentTransform.tx - (oversized ? 20 : 5)}px) translateY(${targetRect.top - ownRect.top + currentTransform.ty - (oversized ? 20 : 5)}px)`;
   },
 
-  _matchWidth($elt, targetRect, ownRect) {
-    return `${$elt.outerWidth() + targetRect.right - targetRect.left - ownRect.right + ownRect.left}px`;
+  _matchWidth($elt, targetRect, ownRect, oversized) {
+    return `${$elt.outerWidth() + targetRect.right - targetRect.left - ownRect.right + ownRect.left + (oversized ? 40 : 10)}px`;
```

Comment:
here we'd use the `expandSizeBy` value rather than the magic number `40` and default to `10`